staging/src/k8s.io/kubectl/pkg/cmd/debug/debug_test.go

```diff
@@ -294,6 +294,27 @@ func TestGenerateDebugContainer(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "netadmin profile",
```

Comment:
nit: I think we need to add unit tests also in `TestGenerateNodeDebugPod` and `TestGeneratePodCopyWithDebugContainer`.

Reply:
added in followup PR: https://github.com/kubernetes/kubernetes/pull/115839